scripts/db-clone/clone-prod-to-staging.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Mother Script: Clone Production to Staging
+# This script orchestrates the cloning of Keycloak and Loculus databases from production to staging
+# Keycloak is dumped second and loaded first to prevent potential race conditions
+
+# Set variables
+CHILD_SCRIPT="./clone.sh"
+PROD_KC_DUMP="production_keycloak_dump.sql"
+PROD_LOC_DUMP="production_loculus_dump.sql"
+PROD_KC_DB="pathoplexus_prod_keycloak"
+PROD_LOC_DB="pathoplexus_prod_loculus"
+STAGING_KC_DB="pathoplexus_staging_keycloak"
+STAGING_LOC_DB="pathoplexus_staging_loculus"
+STAGING_KC_USER="staging_keycloak_user"
+STAGING_LOC_USER="staging_loculus_user"
+
+# Function to check if a command was successful
+check_command() {
+    if [ $? -ne 0 ]; then
+        echo "Error: $1"
+        exit 1
+    fi
+}
+
+# Could screw up columns and values that contain `prod` etc
+# For now not an issue but might eventually want to be more surgical
+perform_sed_replacements() {
+    local file="$1"
+    echo "Performing sed replacements on $file..."
+    sed -i 's/prod/staging/g' "$file"
+    sed -i 's#//pathoplexus.org#//staging.pathoplexus.org#g' "$file"
+    sed -i 's#authentication.pathoplexus.org#authentication-staging.pathoplexus.org#g' "$file"
+    check_command "Failed to perform sed replacements on $file"
```

Comment:
If I understand correctly this will only check the last `sed` command? I kinda think it can be safer to use `set -e` at the top of the script